### DISCUSSION TITLE
Feature Request - Network based webpages

### DISCUSSION THREAD

**Discussion by macj72x** [infinitepower18/Homepage-MobileSafari#7] [General]
Just downloaded this extension and it is working great for me.  I would like to see an option where the extension can point to different sites or disable/enable the extension based on what wifi network you are connected.  My use case for this is, I would like new tabs to go to my personal selfhosted dashboard when I am at home, but do something else when not at my home network. Thanks for making this!

**Comment by infinitepower18** [comment]
Thank you for your suggestion.

Unfortunately, this is currently not possible due to platform limitations.

**Comment by infinitepower18** [comment]
I can however look into implementing a shortcut that lets you set a homepage and then you can do whatever you want with it, such as running the shortcut when the WiFi is changed. Would that work for you?

**Reply by macj72x** [reply]
Yes I think that would work. Thanks for you work on this!

**Comment by infinitepower18** [comment]
@macj72x the shortcut is now available on TestFlight:

https://testflight.apple.com/join/y7e92WCb

Once you installed the latest TestFlight version, hold on the shortcut and create a custom shortcut with a URL of your choice:

<img src="https://github.com/user-attachments/assets/d076a1b4-ea04-4029-87d9-feb7152ceffa" alt="drawing" height="512"/>
<img src="https://github.com/user-attachments/assets/fff03d48-ce1c-4b0f-947d-79e51d3902ee" alt="drawing" height="512"/>

Once done, you can then set up an automation based on WiFi network.

<img src="https://github.com/user-attachments/assets/8d127cfe-b686-4ad6-b11f-37ed7755ac42" alt="drawing" height="512"/>

I will update the App Store version around the time iOS 18 is released.

**Comment by macj72x** [comment]
Thank you!

**Comment by infinitepower18** [comment]
Version 2.1 with the shortcut is now available on the App Store. The TestFlight will no longer be updated.
